docs/pmm/guides/sorts.md

```diff
@@ -0,0 +1,242 @@
+# Sort Options Guide
+
+There are various sort options available depending on what you want to sort.
+
+## ASCII Sort Order
+
+The first thing to note is that Plex Meta Manager utilizes the ASCII sort order, which defines which priority a character gets.
+
+![](images\sorts.jpg)
+
+### Collection Sorting
+
+Collections can be sorted so that they appear in a specific order in the `Collections` tab of your library and within the `Library` tab. This is done using the `sort_title` attribute.
+
+When using the above [ASCII Sort Order](#ascii-sort-order), a collection with a sort title of `!!!My Collection` would appear before a sort title of `!My Collection`, which would be placed before a sort title of `My Collection`
+
+Let's start with an unmodified set of collections which do **not** specify a custom sort order:
+
+???+ example "Example 1 - No Sorting"
+
+    ```yaml
+    collections:
+      Collection 1:
+        plex_all: true
+      Collection 2:
+        plex_all: true
+      Collection 3:
+        plex_all: true
+    ```
+
+    ![](images\sorts_example1.png)
+
+
+Now let's use the `sort_title` attribute to reorder these collections slightly
+
+???+ example "Example 2 - Basic Reorder Sorting"
+
+    ```yaml
+    collections:
+      Collection 1:
+        plex_all: true
+        sort_title: Collection 3
+      Collection 2:
+        plex_all: true
+        sort_title: Collection 1
+      Collection 3:
+        plex_all: true
+        sort_title: Collection 2
+    ```
+
+    ![](images\sorts_example2.png)
+
+    The above could also be achieved using any form of ASCII sort order. This example will produce the same outcome, but uses a (perhaps) illogical set of ASCII characters to define the sort title.
+
+    ```yaml
+    collections:
+      Collection 1:
+        plex_all: true
+        sort_title: "()*Collection 3"
+      Collection 2:
+        plex_all: true
+        sort_title: "!#$Collection 1"
+      Collection 3:
+        plex_all: true
+        sort_title: "%&'Collection 2"
+    ```
+
+You will often see users using sort titles such as `+++++++_My Collection` or `!010_My Collection` in order to force a specific order of collections. Whilst it may not seem logical at a glance, the ASCII sort order allows a lot of fleixibility for power users to be able to tailor the ordering of collections.
+
```

Comment:
I think it's important to make it clear that things like "++++" have absolutely no syntactic meaning.  There's nothing special about '+' other than its position in the standard sort order.